packages/mui-material/src/Autocomplete/Autocomplete.js

```diff
@@ -82,26 +83,20 @@ const AutocompleteRoot = styled('div', {
       { [`& .${autocompleteClasses.input}`]: inputFocused && styles.inputFocused },
       styles.root,
       fullWidth && styles.fullWidth,
+      multiple && styles.multiple,
```

Comment:
This is incorrect. The multiple class isn't applied to the root element.
```suggestion
```

It should be:
```js
{
  [`& .${autocompleteClasses.inputRoot}`]: {
    [`&.${autocompleteClasses.multiple}`]: multiple && styles.multiple,
  },
}
```